packages/vue/src/components/atoms/SfColor/SfColor.stories.js

```diff
@@ -34,7 +34,7 @@ storiesOf("Atoms|Color", module)
           :color="color" 
           :selected="selected" 
           :has-badge="hasBadge"
-          aria-label="color"  
+          :aria-label="color + 'color'"  
```

Comment:
I think here we can use just selected color without 'Color' label, wdyt ?

Reply:
sure, it can be without color label :)